src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -138,6 +139,16 @@ private NeoModule makeSwerveModule(int driveId, int steerId) {
   private void configureBindings() {
     new Trigger(m_driveController::getStartButton)
         .onTrue(new InstantCommand(() -> m_chassis.resetPose(new Pose2d())));
+
+    // The routine automatically stops the motors at the end of the command
+    new Trigger(m_driveController::getAButton)
+        .whileTrue(m_chassis.sysIdQuasistatic(Direction.kForward));
+    new Trigger(m_driveController::getBButton)
+        .whileTrue(m_chassis.sysIdDynamic(Direction.kForward));
+    new Trigger(m_driveController::getXButton)
+        .whileTrue(m_chassis.sysIdQuasistatic(Direction.kReverse));
+    new Trigger(m_driveController::getYButton)
+        .whileTrue(m_chassis.sysIdDynamic(Direction.kReverse));
```

Comment:
We probably don't want these buttons to work on the regular controller. Let's create a new `CommandXboxController` object on port `5` so that we free up these buttons on controllers `0` and `1`